src/environments/environment.common.ts

```diff
@@ -292,7 +292,39 @@ export const environment: GlobalConfig = {
 
     {
       // The default dspace theme
-      name: 'dspace'
+      name: 'dspace',
+      headTags: [
```

Comment:
This new `headTags` setting needs documentation or inline comments. Either add comments here, or add docs to https://wiki.lyrasis.org/display/DSDOC7x/User+Interface+Configuration  (or both).

Currently though, it's not very straightforward what is required in this section, and whether all `<head>` child elements are supported, or just `tagName: link` for now.